### PR TITLE
introduce job.update_meta() to store updated meta to Redis

### PR DESCRIPTION
**Pull request by katichev**
This closes nvie/rq#811

### CODE REVIEW

File: rq/job.py [hunk 1]
```diff
@@ -493,6 +493,16 @@ def save(self, pipeline=None, include_meta=True):
         connection.hmset(key, self.to_dict(include_meta=include_meta))
         self.cleanup(self.ttl, pipeline=connection)
 
+    def update_meta(self):
+        """Stores job meta from the job instance to the corresponding Redis key."""
+        key = self.key
+        meta = dumps(self.meta) if self.meta else None
+        pipeline = self.connection._pipeline()
+        pipeline.hdel(key, 'meta')
```
Comment (selwin) [inline rq/job.py]: I don't think we need "hdel", simply performing "hset" is enough.

Comment (selwin) [inline rq/job.py]: @katichev please update this PR sometime in the next few days. I plan on releasing 0.8.0 to PyPI soon.

File: rq/job.py [hunk 2]
```diff
@@ -493,6 +493,16 @@ def save(self, pipeline=None, include_meta=True):
         connection.hmset(key, self.to_dict(include_meta=include_meta))
         self.cleanup(self.ttl, pipeline=connection)
 
+    def update_meta(self):
```
Comment (selwin) [inline rq/job.py]: Upon further consideration, could you please change the function signature to "save_meta()"? "update_meta()" sounds like it expects an extra argument. Thanks!

Comment (katichev) [inline rq/job.py]: Done, thank you. Indeed, 'update_meta' confused me at first.

**Comment by selwin** [comment]
Thanks!

**Pull request merged**